app/views/hyrax/base/_metadata.html.erb

```diff
@@ -1,7 +1,5 @@
-<dl class="work-show <%= dom_class(presenter) %>" <%= presenter.microdata_type_to_html %>>
```

Comment:
Any reason you couldn't just add `dl.work-show { column-count: 2 }` to the CSS file you changed instead of wrapping everything in an `<ul>` and moving this `<dl>` up?
I think the `itemscope` and `itemtype` properties that `presenter.microdata_type_to_html` adds is important, and breaking up the `<dl>` makes it seem like there's many datalists on the page, when it's one datalist (`<dl>`) with many data terms (`<dt>`) with many data descriptions (`<dd>`)